pennylane/devices/experimental/device_api.py

```diff
@@ -267,11 +267,9 @@ def execute(
 
         **Interface parameters:**
 
-        Note that the parameters contained within the quantum script may contain interface-specific data types, such as
-        ``torch.Tensor`` or ``jax.Array``. If the device does not wish to handle interface-specific parameters, they
-        can implement an optional "internal preprocessing" step that converts all parameters to vanilla numpy. A convenience
-        transform implementing this will be provided. This step allows device to be transparent to things like jitting if they
```

Comment:
if we require numpy-only data in non-backprop executions, perhaps we should still mention `convert_to_numpy_parameters` here to help device developers out?